src/main/java/com/example/demo/model/ReservationRequestModel.java

```diff
@@ -0,0 +1,93 @@
+package com.example.demo.model;
+
+import com.example.demo.core.Pair;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+
+public class ReservationRequestModel {
+
+
+    private Long movieId; 
+    private String name;
+    private String surname;
+    // (seatId,ticketType)
+    private List<Pair<Long, String>> seats;
```

Comment:
może zamiast pair przydałby się lepszy typ żeby nie musieć pisać komentarza jak wyżej?

Reply:
zgodzę się zwiekszyłoby to czytelność kodu, jak i json requestów